schemas/preferences.sublime-settings.json

```diff
@@ -11,7 +11,19 @@
         "color_scheme": {
           "markdownDescription": "Sets the colors used within the text area.",
           "type": "string",
-          "pattern": "^.+\\.((sublime|hidden)-color-scheme|tmTheme)$",
+          "pattern": "^(.+\\.((sublime|hidden)-color-scheme|tmTheme)|auto)",
```

Comment:
Is `$` at the end gone on purpose?

Reply:
I was under the impression that the $ didn't matter, but it does after double checking

<img width="320" alt="Schermafbeelding 2021-05-23 om 23 35 47" src="https://user-images.githubusercontent.com/2431823/119277391-b44ec000-bc1f-11eb-8755-9c8b5c74b3e5.png">

vscode-json-language-server should display a warning here. Let me fix that.